tests/test_visitors/test_ast/test_keywords/test_raise.py

```diff
@@ -112,21 +121,43 @@ def test_raise_good_errors(
     assert_errors(visitor, [])
 
 
-def test_bare_raise(
+@pytest.mark.parametrize(('raise_statement', 'bare_option'), [
+    ("ValueError('1')", True),
+    ("ValueError('1')", False),
+    ('', True),
+])
+def test_bare_raise_correct(
     assert_errors,
     parse_ast_tree,
     default_options,
+    raise_statement,
+    bare_option,
 ):
-    """Testing that bare `raise` is allowed."""
-    code = """
-    try:
-        1 / 0
-    except Exception:
-        raise
-    """
+    """Testing correct instances of `raise SomeException()` and bare `raise`."""
+    code = None
+    if bare_option:
```

Comment:
personally, I dislike using `if` statements in tests, because they make tests harder to read and maintain, so I am suggesting to parameterized function with ready to parse code strings, e.g.:

```python
@pytest.mark.parametrize('code', [
    raise_exception_raw.format('ValueError(1)'),
    raise_exception_with_except_raw.format('ValueError(1)'),
    raise_exception_with_except_raw.format(''),
])
def test_correct raise_statement(
    assert_errors,
    parse_ast_tree,
    default_options,
    code,
 ):
     ...
````